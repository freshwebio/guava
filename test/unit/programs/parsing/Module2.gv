module Order exposing { create, update, delete }