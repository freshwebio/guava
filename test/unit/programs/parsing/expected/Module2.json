{
  "description": "should parse a program that is simply a module definition exposing functions",
  "module": {
    "name": "Order",
    "exposes": {
      "exposed": ["create", "update", "delete"]
    },
    "moduleStatements": {
      "statementList": []
    }
  }
}